pkg/components/util/install.go

```diff
@@ -19,25 +19,24 @@ import (
 	"fmt"
 	"time"
 
-	helmreleaseapi "github.com/fluxcd/helm-controller/api/v2beta1"
 	sourceapi "github.com/fluxcd/source-controller/api/v1beta1"
+	helmrelease "github.com/kinvolk/fluxlib/lib/helm-release"
+	sourcecontroller "github.com/kinvolk/fluxlib/lib/source-controller"
 	"helm.sh/helm/v3/pkg/action"
 	"helm.sh/helm/v3/pkg/chart"
 	"helm.sh/helm/v3/pkg/kube"
 	"helm.sh/helm/v3/pkg/storage/driver"
 	"k8s.io/apimachinery/pkg/api/errors"
 	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
-	"k8s.io/apimachinery/pkg/runtime"
-	"k8s.io/apimachinery/pkg/types"
-	"k8s.io/client-go/tools/clientcmd"
-	"sigs.k8s.io/controller-runtime/pkg/client"
 
 	"github.com/kinvolk/lokomotive/internal"
 	"github.com/kinvolk/lokomotive/pkg/components"
 	"github.com/kinvolk/lokomotive/pkg/k8sutil"
 	"github.com/kinvolk/lokomotive/pkg/version"
 )
 
+var fluxInstallInterval = metav1.Duration{Duration: 5 * time.Minute} //nolint:gomnd
```

Comment:
This can be removed and instead we can use exported variable `components.FluxInstallInterval`